index.js

```diff
@@ -15,11 +15,13 @@
     },
 
     injectCodeFundContainer() {
-      let nav = document.getElementsByClassName('sidebar-nav')
-      let codefundContainer = document.createElement('div')
-      codefundContainer.id = 'codefund'
-      nav[0].insertBefore(codefundContainer, nav[0].firstChild)
-
+      if(document.getElementById("codefund") === null){
```

Comment:
please make the code style consistent to line 25.

Reply:
Hi @njleonzhang, unfortunately`getElementById()` doesn't act like `getElementsByClassName()`, here are the differences :
- `getElementsByClassName()` return an HTMLCollection containing elements
- `getElementById()` returns a single element (because IDs should be unique)

Source: https://www.w3.org/TR/REC-DOM-Level-1/level-one-html.html#method-getElementById

Reply:
@PicchiKevin I mean the indent. 😂

Reply:
woops my bad x)